examples/tutorial/tutorial_notebooks/introduction/exercise_2.ipynb

```diff
@@ -450,8 +450,305 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start = time.time()\n",
+    "\n",
+    "pandas_groupby = pandas_df.groupby(by=\"rounded_trip_distance\").count()\n",
+    "\n",
+    "end = time.time()\n",
+    "pandas_duration = end - start\n",
+    "\n",
+    "print(\"Time to groupby with pandas: {} seconds\".format(round(pandas_duration, 3)))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start = time.time()\n",
+    "\n",
+    "modin_groupby = modin_df.groupby(by=\"rounded_trip_distance\").count()\n",
+    "\n",
+    "end = time.time()\n",
+    "modin_duration = end - start\n",
+    "print(\"Time to groupby with Modin: {} seconds\".format(round(modin_duration, 3)))\n",
+    "\n",
+    "printmd(\"### Modin is {}x faster than pandas at `groupby`!\".format(round(pandas_duration / modin_duration, 2)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Are they equal?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pandas_groupby"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "modin_groupby"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Speed improvements using OmniSci\n",
+    "\n",
+    "Let's repeat the above examples using OmniSci."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import modin.pandas as pd\n",
+    "import pandas as old_pd\n",
+    "import time\n",
+    "import modin.config as cfg\n",
+    "cfg.StorageFormat.put(\"omnisci\")\n",
+    "from IPython.display import Markdown, display\n",
+    "from modin.pandas.test.utils import df_equal\n",
+    "\n",
+    "def printmd(string):\n",
+    "    display(Markdown(string))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "s3_path = \"s3://dask-data/nyc-taxi/2015/yellow_tripdata_2015-01.csv\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#`read_csv` speedups\n",
+    "\n",
+    "start = time.time()\n",
+    "\n",
+    "pandas_df = old_pd.read_csv(s3_path, parse_dates=[\"tpep_pickup_datetime\", \"tpep_dropoff_datetime\"], quoting=3)\n",
+    "\n",
+    "end = time.time()\n",
+    "pandas_duration = end - start\n",
+    "print(\"Time to read with pandas: {} seconds\".format(round(pandas_duration, 3)))\n",
+    "\n",
+    "\n",
+    "start = time.time()\n",
+    "\n",
+    "modin_df = pd.read_csv(s3_path, parse_dates=[\"tpep_pickup_datetime\", \"tpep_dropoff_datetime\"], quoting=3)\n",
```

Comment:
`read_csv` in OmniSci mode doesn't support `quoting` parameter, i beware we won't get speed-up in this way.

Reply:
@amyskov ``read_csv`` correct working with this parameter

Reply:
Yes, you are right, it works correctly, but it uses default pandas implementation, so no speed-up can be observed in this case (so user will see that modin is slower than pandas in the notebook intended to show modin speed-up over pandas).